src/sql/workbench/services/connection/browser/connectionManagementService.ts

```diff
@@ -1231,6 +1231,11 @@ export class ConnectionManagementService extends Disposable implements IConnecti
 						if (connectionMngInfo.serverInfo) {
 							connection.options.isCloud = connectionMngInfo.serverInfo.isCloud;
 						}
+						else if (connection.userName.length === 0) {
+							let connectionDisplayName = this._capabilitiesService.getCapabilities(connectionInfo.providerId).connection.displayName;
```

Comment:
I meant to say the display name for the username option, what you have is the display name for the connection provider.